sieve.py

```diff
@@ -111,10 +111,15 @@ def setup_cluster(project, mode, learn, test_workload, test_config, log_dir, doc
     controllers.deploy[project](docker_repo, docker_tag)
 
     # Wait for project pod ready
-    w = kubernetes.watch.Watch()
-    for event in w.stream(core_v1.list_namespaced_pod, namespace="default", label_selector="sonartag="+project):
-        if event['object'].status.phase == "Running":
-            w.stop()
+    while True:
```

Comment:
It's better to give up eventually rather than loop forever.

Reply:
Will fix this by adding some maximum retrial count.